Source/GitSourceControl/Public/GitSourceControlModule.h

```diff
@@ -103,7 +103,7 @@ class FGitSourceControlModule : public IModuleInterface
 		return GitSourceControlProvider;
 	}
 
-	GITSOURCECONTROL_API static const TArray< FString > & GetEmptyStringArray()
+	static const TArray< FString > & GetEmptyStringArray()
```

Comment:
Why API removed?

Reply:
I don't know how you use it internally, but in my projects, I interact with VCS through ISourceControlModule,  ISourceControlProvider and UEditorAssetLibrary, UEditorAssetSubsystem in my validators and asset action utilities

So exposing such functionality seems a bit wrong architecturally, but again I'm not aware of how you use it in your code, so feel free to revert my changes

At least the GITSOURCECONTROL_API static const TArray< FString > & GetEmptyStringArray() part seems a bit cursed